### PR TITLE
CLOUDP-93396: Restart the entire cluster or a set of processes (ops manager go client)

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-93396

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](e2e/E2E-TESTS.md) (if a new command or e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: internal/cli/opsmanager/clusters/restart.go [hunk 1]
```diff
@@ -71,31 +76,43 @@ func (opts *RestartOpts) Confirm() error {
...
+func RestartBuilder() *cobra.Command {
 	opts := &RestartOpts{}
 	cmd := &cobra.Command{
-		Use:   "restart <name>",
+		Use:   "restart <clusterName>",
 		Short: "Restart a cluster for your project.",
 		Args:  require.ExactArgs(1),
+		Annotations: map[string]string{
+			"args":            "clusterName",
+			"clusterNameDesc": "Name of the cluster that you want to start.",
```
Comment (gssbzn) [inline internal/cli/opsmanager/clusters/restart.go:101]:
```suggestion
			"clusterNameDesc": "Name of the cluster you want to restart.",
```

File: internal/cli/opsmanager/clusters/shutdown.go [hunk 2]
```diff
@@ -95,8 +95,8 @@ func ShutdownBuilder() *cobra.Command {
 		Short: "Shutdown a cluster for your project.",
 		Args:  require.ExactArgs(1),
 		Annotations: map[string]string{
-			"args":     "clusterName",
-			"nameDesc": "Name of the cluster that you want to shutdown.",
+			"args":            "clusterName",
+			"clusterNameDesc": "Name of the cluster that you want to shutdown.",
```
Comment (gssbzn) [inline internal/cli/opsmanager/clusters/shutdown.go:99]: 👀 we may or may not have conflicts here later

File: go.mod [hunk 3]
```diff
@@ -22,6 +22,6 @@ require (
 	github.com/stretchr/testify v1.7.0
 	github.com/tangzero/inflector v1.0.0
 	go.mongodb.org/atlas v0.10.1
-	go.mongodb.org/ops-manager v0.24.0
+	go.mongodb.org/ops-manager v0.24.1-0.20210712085958-0820f6a47c22
```
Comment (gssbzn) [inline go.mod:25]: [non-blocking] feel free to release 0.25

**Review by gssbzn — CHANGES_REQUESTED** [review]
this point is missing
>  I have updated e2e/E2E-TESTS.md (if a new command or e2e test has been added)

**Comment by andreaangiolillo** [comment]
no its already there 
Screenshot 2021-07-12 at 12 52 13

**Comment by andreaangiolillo** [comment]
but I have to change the first two columns 🤦

**Comment by gssbzn** [comment]
>no its already there

weird 🤷‍♂️

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**